worblehat-acceptancetests/src/main/java/de/codecentric/worblehat/acceptancetests/step/page/BookList.java

```diff
@@ -73,6 +73,17 @@ public void booksAreNotBorrowedByBorrower1(String isbns,
         }
     }
 
+    @Then("books $isbns are still borrowed by borrower $borrower")
+    public void booksAreStillBorrowedByBorrower2(String isbns,
+                                                 String borrower2){
+        List<String> isbnList = getListOfItems(isbns);
+        seleniumAdapter.gotoPage(Page.BOOKLIST);
+        HtmlBookList htmlBookList = seleniumAdapter.getTableContent(PageElement.BOOKLIST);
+        for (String isbn : isbnList){
+            assertThat(htmlBookList.getBookByIsbn(isbn).getBorrower(), is(borrower2));
+        }
+    }
+
```

Comment:
Was hat das mit dem Spring Boot Update zu tun?

Reply:
Den Step hatte ich in einem früheren Commit aus versehen rausgelöscht, ist aber notwendig :)